pkg/util/util.go

```diff
@@ -924,7 +927,9 @@ func DownloadFile(url string, filepath string) error {
 	defer out.Close() // #nosec G307
 
 	// Get the data
-	var httpClient = &http.Client{Timeout: HTTPRequestTimeout}
+	var httpClient = &http.Client{Transport: &http.Transport{
+		ResponseHeaderTimeout: HTTPRequestTimeout,
+	}, Timeout: HTTPRequestTimeout}
```

Comment:
Given `ResponseHeaderTimeout` is subset of `Timeout`, can we increase `Timeout` instead of let them share the same timeout? Otherwise we might hit the issue where `ResponseHeaderTimeout` is not reached but `Timeout` is reached.

Reply:
Kept response header timeout at 10s, adjusted the other timeout to 20s